test/helpers/cmd.go

```diff
@@ -33,13 +34,14 @@ import (
 
 // CmdRes contains a variety of data which results from running a command.
 type CmdRes struct {
-	cmd      string        // Command to run
-	params   []string      // Parameters to provide to command
-	stdout   *Buffer       // Stdout from running cmd
-	stderr   *Buffer       // Stderr from running cmd
-	success  bool          // Whether command successfully executed
-	exitcode int           // The exit code of cmd
-	duration time.Duration // Is the representation of the the time that command took to execute.
+	cmd      string          // Command to run
+	params   []string        // Parameters to provide to command
+	stdout   *Buffer         // Stdout from running cmd
+	stderr   *Buffer         // Stderr from running cmd
+	success  bool            // Whether command successfully executed
+	exitcode int             // The exit code of cmd
+	duration time.Duration   // Is the representation of the the time that command took to execute.
+	wg       *sync.WaitGroup // Is used to wait until done when used context execution
```

Comment:
what is "context execution" - does this mean "when used in conjunction with a Context"?